Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt

```diff
@@ -10,19 +10,21 @@ import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingC
 import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.VACCINATION
 
 fun Context.certificateTypesText(certificateTypes: List<String>, separator: String = ", "): String =
```

Comment:
Hi Marc! I am Lukas, and I started on Dec 1st in the CWA Android Team 👋.

A unit test for this extension function that fails before this fix and is successful after this fix would be very useful here I think.

Reply:
Added

Reply:
Hi @LukasLechnerDev! I am Marc, helping the CWA Android Team here and there with fixing bugs ...